plugins/wysiwygarea/plugin.js

```diff
@@ -93,9 +98,82 @@
 						return;
 					}
 
-					editor.editable( new framedWysiwyg( editor, iframe.$.contentWindow.document.body ) );
+					editor.editable( new framedWysiwyg( editor, iframe.getFrameDocument().getBody() ) );
 					editor.setData( editor.getData( 1 ), callback );
+
+					var isIE11 = CKEDITOR.env.ie && CKEDITOR.env.version === 11;
+
+					// (#4462)
+					// Skip IE's below version 11. They don't support MutationObserver.
+					if ( !CKEDITOR.env.ie || isIE11 ) {
+						if ( isIE11 ) {
+							editor.on( 'mode', attachIframeReloader, { iframe: iframe, editor: editor, callback: callback } );
+						}
+
+						observeEditor();
+					}
+				}
+
+				function observeEditor() {
+					mutationObserver = new MutationObserver( function( mutationsList ) {
+						CKEDITOR.tools.array.forEach( mutationsList, verifyIfAddsNodesWithEditor );
+					} );
+
+					mutationObserver.observe( editor.config.observableParent, { childList: true, subtree: true } );
+				}
+
+				function verifyIfAddsNodesWithEditor( mutation ) {
+					if ( mutation.type !== 'childList' || mutation.addedNodes.length === 0 ) {
+						return;
+					}
+
+					CKEDITOR.tools.array.forEach( mutation.addedNodes, checkIfAffectsEditor );
 				}
+
+				function checkIfAffectsEditor( node ) {
+					if ( !node.querySelector ) {
+						return;
+					}
+					var editorSelector = '#' + editor.container.getId(),
+						editorElement = node.querySelector( editorSelector ) || node === editor.container.$;
+
+					if ( !editorElement ) {
+						return;
+					}
+
+					if ( CKEDITOR.env.ie && CKEDITOR.env.version === 11 ) {
+						recreateEditable = true;
+					} else {
+						recreate();
+					}
+				}
+
+				function attachIframeReloader( evt ) {
+					evt && evt.removeListener();
+
+					iframe.on( 'load', function() {
+						if ( recreateEditable ) {
+							recreateEditable = false;
+							recreate();
+						}
+					} );
+				}
```

Comment:
This function is actually invoked before `observeEditor()` etc, so let's move the declaration higher to be easier to find.